APCAppCore/APCAppCore/UI/TabBarControllers/Profile/APCProfileViewController.m

```diff
@@ -114,7 +114,7 @@ - (void)viewDidLoad
     APCUser  *user = appDelegate.dataSubstrate.currentUser;
     self.demographicUploader = [[APCDemographicUploader alloc] initWithUser:user];
     
-    
+    [self refreshView];
```

Comment:
should the [self refreshView] get removed from viewWillAppear in conjuction with this change?